src/op.cpp

```diff
@@ -291,10 +295,10 @@ uint32_t jitc_var_shift(const VarInfo &info, uint32_t index, uint64_t amount) {
 
 // --------------------------------------------------------------------------
 
-template <typename T, enable_if_t<!std::is_signed_v<T>> = 0>
+template <typename T, enable_if_t<!std::is_signed<T>::value> = 0>
```

Comment:
I am curious: why was this change needed? The two should be the same, and the former version was shorter.

Reply:
Specializing `std::is_signed`, `std::is_arithmetic` etc. is officially considered undefined behaviour, which we were using for the `half` type. For GCC `is_signed_v` worked fine, but then I was having issues compiling with Clang which is why I made this change. But then I realised MSVC was also broken and in the end I thought it was safest to define our own `drjit::is_signed` etc. traits to avoid relying on undefined behaviour. So hope that's ok.

Reply:
Making our own specialized version of these classes seems fine and safest.